storage_broker/src/bin/storage_broker.rs

```diff
@@ -438,7 +439,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     // initialize sentry if SENTRY_DSN is provided
     let _sentry_guard = init_sentry(Some(GIT_VERSION.into()), &[]);
     info!("version: {GIT_VERSION}");
-    ::metrics::set_build_info_metric(GIT_VERSION);
+    info!("build_tag: {BUILD_TAG}");
+    ::metrics::set_build_info_metric(GIT_VERSION, BUILD_TAG);
```

Comment:
Does this need the `::` prefix? I cannot see from the `use` what would require it.

Reply:
Yes, we don't need it.
Removed in https://github.com/neondatabase/neon/pull/5576/commits/bdbedec6b2c0da6e5653153e5ec55dc2404cc7dd